src/services/room.service.js

```diff
@@ -0,0 +1,56 @@
+const { ApiError } = require('../exceptions/apiError');
+const { Room } = require('../models/Room');
+
+function create(name) {
+  if (!name) {
+    throw ApiError.badRequest('Name is required');
+  }
+
+  return Room.create({ name });
+}
+
+function join(id) {
+  return Room.findByPk({ id });
```

Comment:
The `join` function uses `findByPk` incorrectly. `findByPk` expects a single primary key value, not an object. Change the call to `Room.findByPk(id)` to correctly retrieve a room by its primary key.